aea/cli/interact.py

```diff
@@ -33,18 +34,34 @@
     DEFAULT_AEA_CONFIG_FILE,
     PackageType,
 )
-from aea.configurations.loader import ConfigLoader
-from aea.connections.stub.connection import (
-    DEFAULT_INPUT_FILE_NAME,
-    DEFAULT_OUTPUT_FILE_NAME,
-    StubConnection,
+from aea.configurations.constants import (
+    DEFAULT_CONNECTION,
+    DEFAULT_PROTOCOL,
+    SIGNING_PROTOCOL,
+    STATE_UPDATE_PROTOCOL,
```

Comment:
would be nice to have one constant to have a set of all default packages,  so maintain easily, package add/load  mechanism can be updated to support the variable amount of components with respect to package type